### PR TITLE
Add Robin to Team Page

### PR DESCRIPTION
**Pull request by robinschaaf**
Add listing for myself to the Team page.

### CODE REVIEW

**Comment by camallen** [comment]
@srallen can @BNewing review this one please?

**Comment by srallen** [comment]
@camallen No prob. Will reassign.

File: app/pages/about/team-page.cjsx [hunk 1]
```diff
@@ -122,6 +122,9 @@ to squeeze a bit of galaxy evolution science in.'''
         bio: '''Becky is an astrophysicist working towards her doctorate in Oxford. She is
         interested in how galaxies change over time and how we can track this evolution.
         Happy-go-lucky about outreach, the Zooniverse and everything.'''
+      robinSchaaf:
+        title: 'Mobile App Developer'
+        bio: '''Robin has a background in web development for educational institutions.  In her free team she enjoys board games and playing bass guitar.'''
```
Comment (simoneduca) [inline app/pages/about/team-page.cjsx]: Sorry, bit too late, but there is a typo. Should be "In her free time" @robinschaaf

Comment (robinschaaf) [inline app/pages/about/team-page.cjsx]: Oh - thanks for the catch @simoneduca !  I'll open a new PR to fix that.

**Pull request merged**